readthedocsext/theme/templates/socialaccount/snippets/provider_list.html

```diff
@@ -14,7 +14,7 @@
         {# Bitbucket doesn't allow more than one callback URL for their OAuth app, so we are redirecting users to the old dashboard for now. #}
         <form class="ui form"
               method="{% if provider.id == "bitbucket_oauth2" %}get{% else %}post{% endif %}"
-              action="{% if provider.id == "bitbucket_oauth2" %}https://{{ PUBLIC_DOMAIN|cut:"app." }}{% endif %}{% provider_login_url provider.id process=process scope=scope auth_params=auth_params %}">
+              action="{% if provider.id == "bitbucket_oauth2" %}https://{{ PRODUCTION_DOMAIN|cut:"app." }}{% endif %}{% provider_login_url provider.id process=process scope=scope auth_params=auth_params %}">
```

Comment:
Oops yeah, we have been bit by this already. We probably should have our own filter for this or something, but we'll not need this for very long either way. `cut` works, I've used `slice` to be more exact though.

```suggestion
              action="{% if provider.id == "bitbucket_oauth2" %}https://{{ PRODUCTION_DOMAIN|slice:"4:" }}{% endif %}{% provider_login_url provider.id process=process scope=scope auth_params=auth_params %}">
```